packages/visual-dev/src/components/Input/rjsf-Input.tsx

```diff
@@ -14,6 +14,7 @@ export function RJSFInput(props: WidgetProps) {
       disabled={props.disabled}
```

Comment:
Would be nice if we could type these change events

Reply:
Agreed, unfortunately, RJSF doesn't type their parameters for `onChange` so I'm not sure we can artificially restrict type at this point, maybe @Locrian24 or @sam-beveridge can weight in.